docs/cli.html

```diff
@@ -381,7 +381,7 @@ <h2 id="convert">convert</h2>
       <blockquote>
 <p>Convert a Visual Builder integration to a CLI integration.</p>
 </blockquote><p><strong>Usage</strong>: <code>zapier convert INTEGRATIONID PATH</code></p><p>The resulting CLI integraiton will be identical to its Visual Builder version and ready to push and use immediately!</p><p>If you re-run this command on an existing directory it will leave existing files alone and not clobber them.</p><p>You&apos;ll need to do a <code>zapier push</code> before the new version is visible in the editor, but otherwise you&apos;re good to go.</p><p><strong>Arguments</strong></p><ul>
-<li>(required) <code>integrationId</code> | To get the integration/app ID, go to &quot;<a href="https://developer.zapier.com//app/developer&quot;">https://developer.zapier.com//app/developer&quot;</a>, click on an integration, and copy the number directly after &quot;/app/&quot; in the URL.</li>
+<li>(required) <code>integrationId</code> | To get the integration/app ID, go to &quot;<a href="https://developer.zapier.com&quot;">https://developer.zapier.com&quot;</a>, click on an integration, and copy the number directly after &quot;/app/&quot; in the URL.</li>
```

Comment:
I'm not sure this info is related to the added feature. Should this and other places it shows up be in a separate PR?

Reply:
@kola-er These are a result of the pre-commit hooks set up for the project. I'm not sure why there were changes - I could try rebasing my branch and seeing if these go away.

Reply:
Just rebased and they went away :)